sf-docker/README.md

```diff
@@ -19,7 +19,7 @@ In order to run SmartFace, you need a valid license.
 ## Samples
 To get up and running as fast as possible, multiple scenarios are available:
 - [`single-camera`](./single-camera/) - a *Simple Setup!* sample
-- [`cloud-matcher`](./cloud-matcher/) - Cloud Matcher deployment sample
+- [`LFIS`](./LFIS/) - Lightweight Facial Identification Service deployment sample
```

Comment:
toto nematchuje update dokumentacie v branchi mkz_cloud_matcher_app.

Reply:
```
### Distribution contents
Once the repository is cloned into your local directory you will see the ```smartface``` folder with a set of files and folders inside. The included folder ```sf-docker``` has several preset use case examples that can be used right away. The examples are organized in folders and each example preset has it's own folder. The contents inside vary. The list of example presets is:
- ```access-control``` - preconfigured for the Access Controll use case
- ```all-in-one``` - an All-in-One setup known from previous version. Contains all available services
- ```lightweight-facial-identification-service``` - a Lightweight Facial Identification Service deployment sample
- ```multi-camera``` - a Multi Camera setup
- ```multi-server``` - sample of the SmartFace distributed on 3 servers
- ```nvidia-jetson```- sample of the SmartFace running on the [Nvidia Jetson](https://developer.nvidia.com/embedded/
jetson-developer-kits) devices
- ```rapid-video-processing``` - sample preset for video processing and investigation
- ```single-camera``` - sample for an easy one camera setup
- ```special``` \ ```jetson-lightweight-facial-identification-service``` - a sample of the Lightweight Facial Identification Service running on the [Nvidia Jetson](https://developer.nvidia.com/embedded/
jetson-developer-kits) devices
- ```special``` \ ```sf-with-keycloak``` - a sample of the SmartFace using Keycloak authentication
```